website/src/main/tut/index.md

```diff
@@ -0,0 +1,26 @@
+---
+layout: home
+title: Home
+---
+
+Scalafix is a rewrite and linting tool for Scala.
```

Comment:
Text lines on the landing page are currently very wide. 

<img width="1038" alt="screen shot 2017-10-10 at 18 14 33" src="https://user-images.githubusercontent.com/1408093/31397675-f267325a-ade6-11e7-8e97-d74a6b71bc32.png">

Would it be possible to increase the font size and create two columns: images/code on one side and title+paragraph on the other? Something like http://frees.io/

Reply:
Increase the font size, sure.

Changing the layout, not so sure. I mean, everything can be done, I just don't know how easy this is with sbt-microsite